### PR TITLE
Remove last of Need API

### PR DESCRIPTION
**Pull request by elenatanasoiu**
Need API has been removed.

For: https://trello.com/c/ClCFMBlG/217-retire-need-api

NB: Depends on https://github.com/alphagov/govuk-puppet/pull/6425 being merged first after which point the current PR will be rebased off master.

### CODE REVIEW

File: modules/govuk/manifests/node/s_api_elasticsearch.pp [hunk 1]
```diff
@@ -65,25 +65,6 @@
       from    => getparam(Govuk_host['calculators-frontend-3'], 'ip'),
       require => Govuk_host['calculators-frontend-3'],
     }
-
-    # FIXME: Remove these firewall rules by moving the need-api app from the backend
-    # machines to the API machines. The MongoDB database will need to move too.
-    # There are more firewall rules in govuk-provisioning that can also be removed.
-    @ufw::allow { 'allow-elasticsearch-http-9200-from-backend-1':
```
Comment (h-lame) [inline modules/govuk/manifests/node/s_api_elasticsearch.pp]: Should this be `ensure: 'absent'`-ed in the previous PR before we remove it outright in this one?

**Comment by h-lame** [comment]
Also, I guess this'll be rebased on master once #6425 is merged?

**Comment by elenatanasoiu** [comment]
Yes it's awaiting rebase. It was added as a step in the checklist and will be fixed up once the first PR has been merged.

File: modules/govuk/manifests/node/s_api_elasticsearch.pp (hunk 1 above)
Comment (elenatanasoiu) [reply modules/govuk/manifests/node/s_api_elasticsearch.pp]: I've added `ensure: absent` in the previous PR https://github.com/alphagov/govuk-puppet/pull/6425/commits/d0eaf7b7e633f0cce572344d7a494ddc43f4b8c4#diff-2f7f03a377299e06b51f7d1b66106db8L72

Comment (elenatanasoiu) [reply modules/govuk/manifests/node/s_api_elasticsearch.pp]: Doesn't look like that works so I will un-do it https://ci.integration.publishing.service.gov.uk/job/govuk-puppet/job/remove-need-api/16/console

**Pull request merged**